### PR TITLE
Update Overview.modified

### PR DESCRIPTION
**Pull request by lemmaa**
### Change Description ###

- Add NN Runtime contents in AI and Machine Learning section

Signed-off-by: Sung-Jae Lee <sj925.lee@samsung.com>

### CODE REVIEW

**Comment by journey2w** [comment]
@asm-foysal
Please review this PR after releasing Tizen Studio 4.6.
The Tizen platform wants to update all docs under the `What is Tizen` menu ASAP.

**Review by asm-foysal — COMMENTED** [review]
@lemmaa hello! The staging in PR was not built at all, I'm closing this PR and reopening it to trigger the stg build.

File: docs/platform/what-is-tizen/overview.md [hunk 1]
```diff
@@ -47,6 +47,32 @@ The most important strengths that enable Tizen to move to the next level are IoT
 
   ![IoTivity for Connectivity](media/about_tizen_6.png)
 
+## AI and Machine Learning
+
```
Comment (arisha-srbd) [inline docs/platform/what-is-tizen/overview.md:51]: "NN Runtime" should be written as a third heading to make it distinguishable from other third headings under the "AI and Machine Learning" section.
Ref: In other PRs (#1615 & #1616), different authors are working on different sections under the "AI and Machine Learning" section.
```suggestion
### NN Runtime
```

File: docs/platform/what-is-tizen/overview.md [hunk 2]
```diff
@@ -47,6 +47,32 @@ The most important strengths that enable Tizen to move to the next level are IoT
...
+   - Parallel executor using CPU and GPU together.
+- Partitioning and multithreading of neural network models at runtime to improve overall inference throughput.
+
+Compiler Toolchain
+
+- Support for interworking with various neural network frameworks and their models.
+   - TensorFlow & TensorFlow lite v1.x & v2.x.
+   - PyTorch & ONNX v1.10.
+- Defining and serving extensible universal container called ‘NN package’.
+   - Accommodating circle(ONE), tflite(TensorFlow lite) model and meta-data in JSON format under directory structure.
```
Comment (arisha-srbd) [inline docs/platform/what-is-tizen/overview.md:71]: Space missing after “circle” and “tflite” and added a comma. 
```suggestion
   - Accommodating circle (ONE), tflite (TensorFlow lite) model, and meta-data in JSON format under directory structure.
```

File: docs/platform/what-is-tizen/overview.md [hunk 3]
```diff
@@ -47,6 +47,32 @@ The most important strengths that enable Tizen to move to the next level are IoT
 
   ![IoTivity for Connectivity](media/about_tizen_6.png)
 
+## AI and Machine Learning
+
+NN Runtime serves as a backend for machine learning APIs for accelerating neural network inference on Tizen devices. It supports heterogeneous computing by combining CPU and GPU, and we plan to expand support to NPU in the near future. It is based on the independent open source project [ONE(On-device Neural Engine](https://github.com/Samsung/ONE), which consists of a runtime virtual machine running on the Tizen device and a compiler toolchain running on the developer's host computer.
```
Comment (arisha-srbd) [inline docs/platform/what-is-tizen/overview.md:52]: A space was missing after "ONE".
```suggestion
NN Runtime serves as a backend for machine learning APIs for accelerating neural network inference on Tizen devices. It supports heterogeneous computing by combining CPU and GPU, and we plan to expand support to NPU in the near future. It is based on the independent open source project [ONE (On-device Neural Engine)](https://github.com/Samsung/ONE), which consists of a runtime virtual machine running on the Tizen device and a compiler toolchain running on the developer's host computer.
```

**Review by arisha-srbd — COMMENTED** [review]
Hello @lemmaa! I have reviewed the modified part of this PR and have made a few minor in-line correction suggestions. Please incorporate the changes. Thank you.

**Review by arisha-srbd — APPROVED** [review]
@lemmaa thank you for incorporating all the changes. It looks good to me. 

@asm-foysal @mijin85cho I am approving this PR for merging.

**Comment by lemmaa** [comment]
> @lemmaa thank you for incorporating all the changes. It looks good to me.

@arisha-srbd , thank you for your collaboration. :)

**Pull request merged**